rpc/wrtc_client.go

```diff
@@ -180,8 +181,15 @@ func dialWebRTC(
 	exchangeCtx, exchangeCancel := context.WithCancel(signalCtx)
 	defer exchangeCancel()
 
+	// atomic "bool" representing whether initial sdp exchange has occured
+	var haveInit int32
```

Comment:
Why not https://pkg.go.dev/sync/atomic#Bool?

Reply:
somehow missed this thanks for pointing out